content.md

```diff
@@ -163,26 +163,26 @@
 
 ## 6. How to clear the application cache
 
-### 6.1 Start from home screen and swipe down from top of screen to display this interface
+### 6.1 Start from home screen and swipe down from top of the screen to display the notification drawer
 ![screenshot](./images/6/1.jpg)
 <div class="pagebreak"></div>
 
-### 6.2 Swipe down once more to display this swipe-down menu
+### 6.2 Swipe down once more to expand this drawer and tap the cog-icon (that looks like a gear), in between the battery life icon and the profile icon
 ![screenshot](./images/6/2.jpg)
 <div class="pagebreak"></div>
 
-### 6.3 Tap *Apps*
+### 6.3 In the Settings menu under the sub-menu *Device* tap *Apps* (situated under *Display* and above *Battery* )
```

Comment:
Highlight *Settings* since it refers to actual text on the screen and remove the '(situated...' bit.